src/Client/LanguageClientOptionsExtensions.cs

```diff
@@ -108,6 +108,18 @@ public static LanguageClientOptions OnStarted(this LanguageClientOptions options
             return options;
         }
 
+        public static LanguageClientOptions UseDefaultRegistrationManager(this LanguageClientOptions options, bool value = true)
```

Comment:
Hmm, feels interesting to have `UseDefaultXYZ` isn't that inferred at the services collection layer by whether or not a consumer registered their own types? Ditto question for the workspace folder manager

Reply:
They may actually not be needed, I'm working through the underlying issue.